include/mbedtls/psa_util.h

```diff
@@ -112,11 +112,16 @@ extern mbedtls_psa_drbg_context_t *const mbedtls_psa_random_state;
  *
  * \param grpid         An Mbed TLS elliptic curve identifier
  *                      (`MBEDTLS_ECP_DP_xxx`).
- * \param[out] bits     On success, the bit size of the curve.
+ * \param[out] bits     On success the bit size of the curve; 0 on failure.
  *
- * \return              The corresponding PSA elliptic curve identifier
+ * \return              On success the corresponding PSA elliptic curve identifier
  *                      (`PSA_ECC_FAMILY_xxx`).
- * \return              \c 0 on failure (\p grpid is not recognized).
+ * \return              \c 0 if \p grpid is not supported.
+ *
+ * \note                A successful conversion means that the curve is supported
+ *                      in PSA. Legacy support (`mbedtls_ecp_xxx`) is only
+ *                      enabled if the curve is builtin (see
+ *                      `config_adjust_legacy_from_psa.h` for details).
```

Comment:
`config_adjust_legacy_from_psa.h` is an internal header, it won't help users. Legacy support is enabled iff `MBEDTLS_ECP_DP_xxx_ENABLED` is defined, but I'm not sure if we need to explain that here. Isn't it enough to just say

> If the curve is supported in the PSA API, this function succeeds, even if the curve is not supported by the ECP module. If the curve is not supported in the PSA API, this function returns 0.

Maybe we even don't need this note: we could just clarify it in the `return` statement above (replace “on success” by what causes success).